neptune/internal/backends/alpha_integration_backend.py

```diff
@@ -91,21 +152,20 @@ def create_experiment(self,
             "remotes": git_info.remote_urls
         } if git_info else None
 
+        api_params = {
+            "projectIdentifier": str(project.internal_id),
+            "cliVersion": self.client_lib_version,
+            "gitInfo": git_info,
+            "customId": name,
+        }
+
+        kwargs = {
+            'experimentCreationParams': api_params,
+            'X-Neptune-CliVersion': self.client_lib_version,
+        }
+
         try:
-            params = {
-                "projectIdentifier": str(project.internal_id),
-                "cliVersion": self.client_lib_version,
-                "gitInfo": git_info,
-                "customId": name,
-            }
-
-            kwargs = {
-                'experimentCreationParams': params,
-                'X-Neptune-CliVersion': self.client_lib_version,
-            }
             api_experiment = self.leaderboard_swagger_client.api.createExperiment(**kwargs).response().result
-
-            return self._convert_to_experiment(api_experiment, project)
         except HTTPNotFound:
```

Comment:
Does alpha backend use the same exceptions for quota?

Reply:
I'm not sure if it even recognize quota limit. Verify it and if it's not supported and task to jira.

Reply:
Ok, I'll check that

Reply:
Simplified to extent used in alpha when creating new experiment.